.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,18 @@
-
-
 <!--
 Thank you very much for contributing code or documentation to restic! Please
 fill out the following questions to make it easier for us to review your
 changes.
-
-You do not need to check all the boxes below all at once, feel free to take
-your time and add more commits. If you're done and ready for review, please
-check the last box.
 -->
 
-What does this PR change? What problem does it solve?
------------------------------------------------------
+What changes are made in this PR? What problem does it solve?
+-------------------------------------------------------------
 
 <!--
 Describe the changes and their purpose here, as detailed as needed.
 -->
 
-Was the change discussed in an issue or in the forum before?
-------------------------------------------------------------
+Was the change previously discussed in an issue or on the forum?
```

Comment:
Is it really not "in" the forum? 👀

Reply:
Since the forum is just a website, users post _on_ the forum (_on_ the website).

Reply:
This seems awfully wrong to me, considering a forum is something you are *in* even if the forum happens to be on some platform that you are normally *on* (like a house being on the ground doesn't mean you are on the house - you're still in the house). But I will digress and accept your suggestion - I presume you are more native english speaking than I am.